content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/used.rb

```diff
@@ -6,7 +6,7 @@ def consumption(source)
   {
     :cpu                 => source.allocated_vcpu,
     :memory              => source.allocated_memory,
-    :vms                 => source.vms.count { |vm| vm.id unless vm.archived },
+    :vms                 => source.vms.count { |vm| vm.id unless vm.archived || vm.orphaned },
```

Comment:
using `if vm.ems_id` makes me and is a bit cleaner with less indirection. I'll make the change.